README.md

```diff
@@ -228,16 +236,58 @@ if (_.contains(projectsInRGSoC, {
 
 [![Relative date](https://img.shields.io/date/1577392258?color=important&label=started&logo=github)](https://github.com/vinitshahdeo/) [![Maintenance](https://img.shields.io/maintenance/yes/2020?color=green&logo=github)](https://github.com/vinitshahdeo/)
 
-<a href="https://fayz.in/stories/s/1522/0/?ckt_id=ZGL1ZGVk&title=story_of_vinit_shahdeo"><img src="assets/vinit-shahdeo.jpg" width="150px" height="150px" /></a>
-
-**[Vinit Shahdeo](https://www.linkedin.com/in/vinitshahdeo/)**                                                                                                                                        
- <a href="https://twitter.com/Vinit_Shahdeo"><img src="assets/twitter.png" width="32px" height="32px"></a> <a href="https://www.facebook.com/vinit.shahdeo"><img src="assets/facebook.png" width="32px" height="32px"></a> <a href="https://www.linkedin.com/in/vinitshahdeo/"><img src="assets/linkedin.png" width="32px" height="32px"></a> 
+|                                                                                         <a href="https://fayz.in/stories/s/1522/0/?ckt_id=ZGL1ZGVk&title=story_of_vinit_shahdeo"><img src="assets/vinit-shahdeo.jpg" width=150px height=150px /></a>                                                                                         |
```

Comment:
`width="150px" height="150px"`
Please put double quotes otherwise style wont work after deployment.

Reply:
Done. This is how it looks:
<img width="651" alt="Screenshot 2020-05-20 at 16 25 00" src="https://user-images.githubusercontent.com/11659160/82458300-ccdd4500-9ab6-11ea-84e3-183010bda641.png">

@vinitshahdeo Kindly suggest.
Thanks